mobility_data/importers/bicycle_stands.py

```diff
@@ -10,13 +10,17 @@
 
 from .utils import (
     delete_mobile_units,
-    get_closest_street_name,
+    get_closest_address_full_name,
     get_municipality_name,
     get_or_create_content_type,
-    get_street_name_translations,
     set_translated_field,
 )
 
+NAME_PREFIX = {
+    "fi": "Pyöräpysäköinti",
```

Comment:
Should really use final string variables as keys. Always

Reply:
Fixed.